cmd/juju/secrets/list_test.go

```diff
@@ -47,17 +49,22 @@ func (s *ListSuite) TestListTabular(c *gc.C) {
 	s.secretsAPI.EXPECT().ListSecrets(false).Return(
 		[]apisecrets.SecretDetails{{
 			Metadata: coresecrets.SecretMetadata{
-				ID: 666, Scope: coresecrets.ScopeApplication,
+				ID: 666, RotateDuration: time.Hour,
 				Revision: 2, Path: "app.password", Provider: "juju"},
+		}, {
+			Metadata: coresecrets.SecretMetadata{
+				ID:       667,
+				Revision: 1, Path: "app.apitoken", Provider: "juju"},
 		}}, nil)
 	s.secretsAPI.EXPECT().Close().Return(nil)
 
 	ctx, err := cmdtesting.RunCommand(c, secrets.NewListCommandForTest(s.store, s.secretsAPI))
 	c.Assert(err, jc.ErrorIsNil)
 	out := cmdtesting.Stdout(ctx)
 	c.Assert(out, gc.Equals, `
-ID         Scope  Revision  Backend  Path          Age
-666  application  2         juju     app.password  0001-01-01  
+ID   Revision  Rotate  Backend  Path          Age
+666         2  1 hour  juju     app.password  0001-01-01  
```

Comment:
Age implies a duration, maybe we can use the new humanizeDuration or change the title.

Reply:
Age is formatted with our existing `UserFriendlyDuration` func, which is for printing the difference between 2 times, one of them being `now`.
The Rotate value is an interval, every X hours etc, which is not quite the same.